### DISCUSSION TITLE
[Bug]: HTML content not available in object views

### DISCUSSION THREAD

**Discussion by DerickVanDerSandt** [pimcore/pimcore#12677] [General]
### Expected behavior

The query table must contain HTML data.

### Actual behavior

The query table removes HTML data.

### Steps to reproduce

We currently have an object that uses the WYSIWYG controls. When viewing the data in the object store table we can see the HTML.
If we look at the corresponding query table those fields have HTML removed. When we check the view we can see that the view is based on the object_query table. As such when we query the view (in order to maintain inheritance and also not have to query relational tables) we are not able to get the HTML that was saved in the store table, this in turn means we have to join the view against the store twice (once for object and once for parent) in order to get this field.
Does anyone know why this has been implemented this way, or if it's an oversite or a workaround?
This is for pimcore 6.6.11

**Comment by fashxp** [comment]
Pimcore 6.6 is EOL ... moved to discussions.

**Reply by DerickVanDerSandt** [reply]
I did initially load it as a discussion but nobody came back to us. Just trying to find out if this was done for design purposes or if its an issue. I am not looking for it to be resolved and secondly is this the case in Pimcore X?

**Reply by fashxp** [reply]
I guess this was done to not uselessly crowed the query table as it is used for querying data only and there HTML code does not make any sense.

**Reply by SimeonAngel** [reply]
so basically what you're saying is that if you have an html field and you need to extract that information you have to query the store table then having to join against itself in order to maintaine inhertiance, we would then also have to join this against the view in order to get the joining information (the purpose of the view).  In essence if one uses the WYSIWYG field and you want to get the data out you have to do a whole lot of heavy lifting

**Reply by fashxp** [reply]
query table is not meant for extracting information but for querying data only. 
if you want to extract information (also considering data inheritance etc.) please use the API.

**Reply by SimeonAngel** [reply]
So there are many issues with using the API's.
Pimcore advises to use DataHub because the API's are going to be depricated, as such we used the datahub, however we had the following issues

1. To extract the amount of data we are trying to extract is extremly slow.
2. GraphQL does not support nested blocks

Either which way in the documentation for v6.6 it says
![image](https://user-images.githubusercontent.com/63024167/179544837-e2fe4d54-04c0-406a-abfc-6bdff4e9ee49.png)

That the object view is a join on the object_store and objects table.
However if we look at the view, the view infact is a join on the query table, in the docs it also says the object_view table is used to retreive data including inheritance etc.

From here it would imply that I should be able to get the HTML fields out of the views, however because it is joined on the query table and not the store table, one cannot get this info without doing the joins as mentioned above
